packages/conf-gmp/conf-gmp.1/opam

```diff
@@ -6,7 +6,10 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "GPL"
 build: [
-  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"]
+  ["opamconfig" "-d" "gmp:" "-f" "./conf-gmp.config" "sh" "-exc" "cc -c ${CFLAGS} test.c"]
+]
+depends: [
+  "opamconfig" {build & >= "0.3.0"}
```

Comment:
It looks like the Alpine musl support has regressed -- the other distros all pass CI.

```
+ opamconfig "-d" "gmp:" "-f" "./conf-gmp.config" "sh" "-exc" "cc -c ${CFLAGS} test.c" (CWD=/home/opam/.opam/4.03.0/build/conf-gmp.1)
[ERROR] The compilation of conf-gmp failed at "opamconfig -d gmp: -f
        ./conf-gmp.config sh -exc cc -c ${CFLAGS} test.c".
```